app/src/main/java/org/simple/clinic/instantsearch/InstantSearchEffectHandler.kt

```diff
@@ -48,6 +48,7 @@ class InstantSearchEffectHandler @AssistedInject constructor(
       .addConsumer(OpenShortCodeSearchScreen::class.java, ::openShortCodeSearch, schedulers.ui())
       .addAction(OpenQrCodeScanner::class.java, uiActions::openQrCodeScanner, schedulers.ui())
       .addTransformer(CheckIfPatientAlreadyHasAnExistingNHID::class.java, checkIfPatientAlreadyHasAnExistingNHID(schedulers.ui()))
+      .addConsumer(ShowNHIDErrorDialog::class.java, { uiActions.showNHIDErrorDialog() }, schedulers.ui())
```

Comment:
A suggestion, we can make this easier to read by changing the lambda to a method reference.